pepdbagent/modules/project.py

```diff
@@ -126,14 +118,42 @@ def get(
         except NoResultFound:
             raise ProjectNotFoundError
 
+    def _get_samples(self, session: Session, prj_id: int, with_id: bool) -> List[Dict]:
+        """
+        Get samples from the project. This method is used to retrieve samples from the project,
+            with open session object.
+
+        :param session: open session object
+        :param prj_id: project id
+        :param with_id: retrieve sample with id
+        """
+        samples_results = session.scalars(select(Samples).where(Samples.project_id == prj_id))
+        result_dict = {}
+        for sample in samples_results:
+            sample_dict = sample.sample
+            if with_id:
+                sample_dict["ph_id"] = sample.guid
```

Comment:
Should this be a constant? And will this be confusing in the future?

Reply:
Like, should we just always call it a `guid`?

Reply:
Yeah, I missed it